a/arnah.md

```diff
@@ -0,0 +1,12 @@
+# Arnah
+
+
+![Arnah](arnah.jpg)
```

Comment:
This is lowercase, correct.
But the filename of the image has a capital letter!